src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -133,8 +133,9 @@ internal View(global::System.IntPtr cPtr, bool cMemoryOwn, bool shown = true) :
                 PositionUsesPivotPoint = false;
             }
 
-            _onWindowSendEventCallback = SendViewAddedEventToWindow;
-            this.OnWindowSignal().Connect(_onWindowSendEventCallback);
+            //ToDo: this has memory leak and this is not used currently. will be fixed soon by using Event subscribing pattern.
```

Comment:
I remember that I made this event addition request to support automatic test in appium. Is there any problem using appium even if I delete it?

Reply:
You are right, there would be a problem. 
I will upload patch again soon.

Reply:
Is there any update for this?

Can i remove `View._onWindowSendEventCallback`?
There is build warning.
`/home/yohoho/tizen/TizenFX/src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs(61,43): warning CS0649: Field 'View._onWindowSendEventCallback' is never assigned to, and will always have its default value null [/home/yohoho/tizen/TizenFX/src/Tizen.NUI/Tizen.NUI.csproj]`